app/src/main/java/com/github/se/polyfit/ui/screen/LoginScreen.kt

```diff
@@ -58,43 +66,60 @@ fun LoginScreen(navController: Navigation) {
-              Spacer(Modifier.weight(0.15f))
+              Spacer(Modifier.weight(0.2f))
+
+              Title(Modifier.testTag("LoginTitle"), 50.sp)
+
+              Spacer(Modifier.weight(0.6f))
+
+              SignInButton {
+                Log.d("LoginScreen", "button clicked")
+                createSignInIntent()
+              }
+
+              Spacer(Modifier.weight(0.03f))
 
               Text(
-                  text = "Welcome",
-                  fontSize = 30.sp,
-                  modifier = Modifier.testTag("LoginTitle"),
-              )
-
-              Spacer(Modifier.weight(0.5f))
-
-              OutlinedButton(
-                  onClick = {
-                    Log.d("LoginScreen", "button clicked")
-                    createSignInIntent()
-                  },
-                  shape = RoundedCornerShape(20.dp),
-                  colors =
-                      ButtonDefaults.buttonColors(
-                          contentColor = Color.Black, containerColor = Color.White),
-                  modifier = Modifier.testTag("LoginButton")) {
-                    val imageModifierGoogle = Modifier.size(24.dp)
-
-                    Image(
-                        painter = painterResource(id = R.drawable.google_logo),
-                        contentDescription = "Google_logo",
-                        contentScale = ContentScale.Fit,
-                        modifier = imageModifierGoogle)
-                    Text(text = "Sign in with Google", modifier = Modifier.padding(start = 16.dp))
-                  }
-              Spacer(Modifier.weight(0.3f))
+                  termText,
+                  textAlign = TextAlign.Center,
+                  modifier = Modifier.testTag("TermsOfService"))
+
+              Spacer(Modifier.weight(0.2f))
             }
       }
 }
 
+val termText = buildAnnotatedString {
+  append("By clicking continue, you agree to our \n")
+  withStyle(style = SpanStyle(fontWeight = FontWeight.Bold)) { append("Terms of Service") }
+  append(" and ")
+
+  withStyle(style = SpanStyle(fontWeight = FontWeight.Bold)) { append("Privacy Policy") }
+}
+
+@Composable
+fun SignInButton(onClick: () -> Unit) {
+
+  Button(
+      onClick = { onClick() },
+      shape = RoundedCornerShape(20.dp),
+      colors =
+          ButtonDefaults.buttonColors(
+              contentColor = MaterialTheme.colorScheme.onPrimary, containerColor = primaryPurple),
+      modifier = Modifier.testTag("LoginButton")) {
+        val imageModifierGoogle = Modifier.size(24.dp).absoluteOffset(x = (-13).dp, y = 0.dp)
+
+        Image(
+            painter = painterResource(id = R.drawable.google_logo),
+            contentDescription = "Google_logo",
+            contentScale = ContentScale.Fit,
+            modifier = imageModifierGoogle)
+        Text(text = "Sign in with Google")
+      }
+}
+
 class Authentication(private val navigation: Navigation, private val context: Context) {
```

Comment:
If we try to follow the view, viewmodel, model pattern, the Authentification class should belong its own class in the model folder. This  would improve the readability of the code